### PR TITLE
Add getCollectionDetails function

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: packages/libs/api/sdk/README.md [hunk 1]
```diff
@@ -111,6 +111,24 @@ client.nft
...
+| address  | string | ❌       | Contract address of NFT | 0x2106C00Ac7dA0A3430aE667879139E832307AeAa |
+
+```ts
+import { QuickNodeSDK } from '@quicknode/sdk';
+
+const client = new QuickNodeSDK();
+
+client.nft
+  .getCollectionDetails({
+    address: '0x2106C00Ac7dA0A3430aE667879139E832307AeAa',
```
Comment (AlejandroFabianCampos) [inline packages/libs/api/sdk/README.md:127]: until we fix the address normalization, we should suggest that they use all lowercase?

**Review by AlejandroFabianCampos — APPROVED** [review]
Just this small note, otherwise :shipit:

File: packages/libs/api/sdk/README.md (hunk 1 above)
Comment (johnpmitsch) [reply packages/libs/api/sdk/README.md:127]: I think once we know more about how the casing works on the icy side, we can update our docs as needed

Comment (AlejandroFabianCampos) [reply packages/libs/api/sdk/README.md:127]: makes sense 👍

**Pull request merged**